x-pack/plugins/uptime/public/components/common/header/__tests__/page_header.test.tsx

```diff
@@ -35,4 +58,46 @@ describe('PageHeader', () => {
     );
     expect(component).toMatchSnapshot('page_header_with_extra_links');
   });
+
+  it('renders null when on a step detail page', () => {
+    const history = createMemoryHistory();
+    // navigate to step page
+    history.push('/journey/1/step/1');
+    const component = renderWithRouter(
+      <MountWithReduxProvider store={state}>
+        <PageHeader />
+      </MountWithReduxProvider>,
+      history
+    );
+    expect(component.html()).toBe(null);
+  });
+
+  it('renders monitor header without tabs when on a monitor page', () => {
+    const history = createMemoryHistory();
+    // navigate to monitor page
+    history.push('/monitor/1');
+    const component = renderWithRouter(
+      <MountWithReduxProvider store={state}>
+        <PageHeader />
+      </MountWithReduxProvider>,
+      history
+    );
+    expect(component.find('h1').text()).toBe(monitorName);
+    expect(component.find('[data-test-subj="uptimeSettingsToOverviewLink"]').length).toBe(0);
```

Comment:
Is there a way we can just add a `data-test-subj` to the whole tab group instead of looking for specific links? Seems like the wrong approach, and fragile if the links themselves change.